scripts/firewall/vyatta-firewall.pl

```diff
@@ -293,7 +293,7 @@ sub write_refcnt_file {
         print $FILE join("\n", @lines), "\n";
         close($FILE);
     } else {
-        system("rm $refcnt_file");
+        system("[ -e $refcnt_file ] && rm $refcnt_file");
```

Comment:
Let's not overuse `system()`. Built-in functions are always faster: `unlink($refcnt_file) if -e $refcnt_file`.

Reply:
@dmbaturin  Done